blocks/melody/_shared/mixins.less

```diff
@@ -114,11 +114,37 @@
     }
 }
 
-
+// Calculating contrasts from here: http://24ways.org/2010/calculating-color-contrast/
+.colour-guard(@colour, @dark: #000000, @light: #ffffff) when ((((red(@colour)*299)+(green(@colour)*587)+(blue(@colour)*114))/1000) > 204) {
+    color: @dark;
+}
+.colour-guard(@colour, @dark: #000000, @light: #ffffff) when ((((red(@colour)*299)+(green(@colour)*587)+(blue(@colour)*114))/1000) =< 204) {
+    color: @light;
+}
+.colour-guard-svg(@colour, @dark: #000000, @light: #ffffff) when ((((red(@colour)*299)+(green(@colour)*587)+(blue(@colour)*114))/1000) > 204) {
+    fill: @dark;
+}
+.colour-guard-svg(@colour, @dark: #000000, @light: #ffffff) when ((((red(@colour)*299)+(green(@colour)*587)+(blue(@colour)*114))/1000) =< 204) {
+    fill: @light;
+}
 
 #ff_mix-sections {
```

Comment:
Is this used anywhere aside from the container-landing sections? If not it should probably just live inside the container-landing less file.